tools/AzureRM.Netcore/AzureRM.Netcore.psm1

```diff
@@ -0,0 +1,25 @@
+Write-Warning "AzureRM.Netcore has been deprecated.  Use the 'Az' module instead.  The 'Az' module is avalable from the PSGallery https://www.powershellgallery.com/packages/Az/. You can find information about getting started with 'Az' at https://docs.microsoft.com/en-us/powershell/azure/new-azureps-module-az. To uninstall AzureRM.Netcore you can use the provided 'Uninstall-AzureRMNetcore' cmdlet."
+
+function Uninstall-AzureRMNetcore {
+    [CmdletBinding(SupportsShouldProcess=$true)]
+    param()
+    begin {}
+    process{ 
+        $allModules = (Get-Module -ListAvailable)
```

Comment:
@markcowl nit: I'm not sure if it helps that much with performance, but we can add an extra filter to this `Get-Module` call to only get the Netcore modules:

```powershell
$allModules = (Get-Module -Name Azure*.Netcore -ListAvailable)
```